app/pages/nics/Class4ExemptionCategoryPage.scala

```diff
@@ -24,23 +24,21 @@ import models.database.UserAnswers
 import models.journeys.nics.ExemptionCategory
 import models.journeys.nics.ExemptionCategory.{DiverDivingInstructor, TrusteeExecutorAdmin}
 import play.api.mvc.Call
-import queries.Settable
-
-import scala.collection.immutable._
 
 case object Class4ExemptionCategoryPage extends NicsBasePage[ExemptionCategory] {
   override def toString: String = "class4ExemptionCategory"
 
   override def nextPageInNormalMode(userAnswers: UserAnswers, businessId: BusinessId, taxYear: TaxYear): Call =
     userAnswers.get(this, businessId) match {
-      case Some(TrusteeExecutorAdmin)  => routes.Class4NonDivingExemptController.onPageLoad(taxYear, NormalMode)
       case Some(DiverDivingInstructor) => routes.Class4DivingExemptController.onPageLoad(taxYear, NormalMode)
+      case Some(TrusteeExecutorAdmin)  => routes.Class4NonDivingExemptController.onPageLoad(taxYear, NormalMode)
       case None                        => standard.routes.JourneyRecoveryController.onPageLoad()
     }
 
   override def hasAllFurtherAnswers(businessId: BusinessId, userAnswers: UserAnswers): Boolean =
-    userAnswers.get(this, businessId).isDefined
-
-  override val dependentPagesWhenAnswerChanges: List[Settable[_]] =
```

Comment:
Clearing dependent pages will need to happen in the controller onSubmit method instead.